server/src/test/java/com/blackship/battlesheep/game/state/board/BoardModifierTest.java

```diff
@@ -0,0 +1,67 @@
+package com.blackship.battlesheep.game.state.board;
+
+import com.blackship.battlesheep.game.FieldState;
+import com.blackship.battlesheep.game.Position;
+import com.blackship.battlesheep.game.state.fleet.Ship;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * @author Anna Gawda
+ * @since 26.07.17
+ */
+@Test
+public class BoardModifierTest {
+
+    private Board board;
+
+    @DataProvider
+    private Object[][] boardSheepData() {
+        return new Object[][] {
+                {new ArrayList<>(Arrays.asList(1, 11, 12, 13))},
+                {new ArrayList<>(Arrays.asList(1, 11 , 21, 31))}
+        };
+    }
+
+    @DataProvider
+    private Object[][] boardSheepsData() {
```

Comment:
Sheeps? Not ships? 
![6738204-shaun-the-sheep-the-movie-643-643](https://user-images.githubusercontent.com/18446697/28667709-f1328628-72cc-11e7-83c4-077a5941d6d5.jpg)